### ISSUE TITLE
Purchase Sonar Sensors

### ISSUE THREAD

**Issue by ZefraSK** [Weber-State-Submarine-Project/Submarine#15]
DONE: When Acquired Sonar Sensors

CONSIDER:
- Lets buy one for now

TASK:
- [x]  Buy 1x sonar
- [x]  Buy 1x UART to USB
- [ ]  Determine if usable

**Comment by ZefraSK** [comment]
Waiting on Judy's Response

**Comment by ZefraSK** [comment]
Judy says to talk Dr. West

**Comment by ZefraSK** [comment]
Buy Sonar and UART to USB Personally

**Comment by ZefraSK** [comment]
Ordered Both: 

- $46 Sonar (Shipping sucks...)
- $11 UART to USB

**Comment by ZefraSK** [comment]
Sonar and UART to USB received:

New Task:
- [x] Hand to Ben
